tests/client_tests/node-postgres/tests/test_cursors.js

```diff
@@ -0,0 +1,28 @@
+const cratedb = require.main.require('db');
+const Cursor = require("pg-cursor");
+
+const chai = require("chai");
+const assert = chai.assert;
+
+
+async function run() {
+  const client = await cratedb.connect();
+  const stmt = "SELECT * FROM generate_series(1, 35)";
+  const cursor = client.query(new Cursor(stmt));
+
+  let res1 = await cursor.read(5);
+  let res2 = await cursor.read(10);
+  let res3 = await cursor.read(10);
```

Comment:
maybe another read, and then test that the result set is exhausted?

Reply:
Added a second test case